app/javascript/components/pages/dashboard/container.spec.js

```diff
@@ -0,0 +1,68 @@
+import { fromJS } from "immutable";
+
+import { mountedComponent, screen } from "../../../test-utils";
+
+import Dashboard from "./container";
+
+describe("<Dashboard />", () => {
+
+  const initialState = fromJS({
+    user: {
+      reportingLocationConfig: {
+        field_key: "owned_by_location",
+        admin_level: 2,
+        admin_level_map: { 1: ["province"], 2: ["district"] },
+        label_keys: ["district"]
+      }
+    }
+  });
+
+  it("should render a <PageContainer /> component", () => {
+    mountedComponent(<Dashboard />, {}, initialState);
+    expect(screen.getAllByTestId("page-heading")).toHaveLength(1);
+  });
+  it("should render a <PageHeading /> component", () => {
+    mountedComponent(<Dashboard />, {}, initialState);
+    expect(screen.getByText(/navigation.home/i)).toBeInTheDocument();
+  });
+  it("should render a <PageContent /> component", () => {
+    mountedComponent(<Dashboard />, {}, initialState);
+    expect(screen.getByTestId("page-heading")).toBeInTheDocument();
+  });
+  it("should render a <Overview /> component", () => {
+    mountedComponent(<Dashboard />, {}, initialState);
+    expect(screen.getByText(/navigation.home/i)).toBeInTheDocument();
+  });
+  it("should render a <SharedFromMyTeam /> component", () => {
+    mountedComponent(<Dashboard />, {}, initialState);
+    expect(screen.getByTestId("page-heading")).toBeInTheDocument();
```

Comment:
This doesn't tell us what dashboard was rendered, either try to find the title of each dashboard, for example, this would be: `dashboard.dash_shared_with_my_team` or try to add a `testId` to each dashboard.

Reply:
Done